src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1229,3 +1230,42 @@ def _configure_security_policy_storage_params(arg_ctx):
                    id_part='name')
 
         c.argument('resource_group_name', arg_type=resource_group_name_type)
+
+    ###############################################
+    #             sql instance failover-group     #
+    ###############################################
+
+    with self.argument_context('sql instance-failover-group') as c:
+        c.argument('instance_failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
+
+        c.argument('failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
+
+        c.argument('managed_instance',
+                   arg_type=managed_instance_param_type,
+                   options_list=['-s'])
+
+        c.argument('partner_managed_instance',
+                   help="The name of the partner managed instance of a Instance Failover Group")
+
+        c.argument('partner_resource_group',
+                   help="The name of the resource group of the partner managed instance")
+
+        c.argument('failover_policy',
+                   help="The failover policy of the Instance Failover Group",
+                   arg_type=get_enum_type(InstanceFailoverPolicyType))
+
+        c.argument('grace_period',
+                   help='Interval in hours before automatic failover is initiated '
```

Comment:
Same comment, extract commonality between mi and standalone db FG. Don't copy paste!!